prisma/migrations/20220726190348_init/migration.sql

```diff
@@ -0,0 +1,42 @@
+-- CreateTable
+CREATE TABLE "Post" (
+    "id" SERIAL NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Would usually prefer DATETIME over TIMESTAMP, it makes data more readily accessible and use a trigger

eg

`createdAt` DATETIME NULL DEFAULT CURRENT_TIMESTAMP,
`updatedAt` DATETIME NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,

Would also prefer `modifiedAt` over `updatedAt`

Reply:
Ah these were the default migrations added prisma (removed them in further migrations) - DateTime or the lovely DateTimeOffset